Profile Folder/chrome/locale/ja/dtd/gwizard.dtd

```diff
@@ -1,15 +1,15 @@
-<!ENTITY gwizard.chooseAVisualStyle	"Choose a visual style">
-<!ENTITY gwizard.additionalSettings	"Additional settings">
-<!ENTITY gwizard.finish	"Finish">
+<!ENTITY gwizard.chooseAVisualStyle	"外観を選ぶ">
+<!ENTITY gwizard.additionalSettings	"追加の設定">
+<!ENTITY gwizard.finish	"完了">
 
-<!ENTITY gwizard.gsettingsDescription 		"Make your Geckium truly unique.">
-<!ENTITY gwizard.launch 					"Launch">
-<!ENTITY gwizard.chromiumFlagsTitle			"Chromium Flags">
-<!ENTITY gwizard.chromiumFlagsDescription	'Chromium Flags ported over to Geckium. Available in "about:flags" or Geckium Settings.'>
+<!ENTITY gwizard.gsettingsDescription 		"あなただけの Geckium にカスタマイズする">
+<!ENTITY gwizard.launch 					"起動する">
+<!ENTITY gwizard.chromiumFlagsTitle			"Chromium の試験的機能">
+<!ENTITY gwizard.chromiumFlagsDescription	'Geckium には Chromium で利用できる試験的機能がポートされています。 「about:flags」ページ、または Geckium の設定 で変更できます。'>
 
-<!ENTITY gwizard.YouAreAllSetTitle			"You're all set!">
-<!ENTITY gwizard.YouAreAllSetDescription	'You can reach Geckium Settings at any time by clicking the Geckium Settings toolbar button or by right clicking the toolbar.'>
+<!ENTITY gwizard.YouAreAllSetTitle			"初回設定は以上です！いってらっしゃい！">
+<!ENTITY gwizard.YouAreAllSetDescription	'「Geckium の設定」ページへ移動するには、ツールバー内の「Geckium の設定」ボタンを押すか、ツールバーを右クリックして出てくるメニューにある「Geckium の設定」を押してください。'>
 
-<!ENTITY gwizard.publicBetaTitle			"Geckium is an unstable, prerelease theme">
-<!ENTITY gwizard.publicBetaDescription	    "This is the Public Beta of Geckium, and so significant inaccuracies, incomplete localisations, or random critical bugs could be encountered - if you find any bugs, please make sure to report them on the Geckium GitHub page that you downloaded Geckium from.">
-<!ENTITY gwizard.publicBetaLocalise         "If you are fluent with non-English languages, you can also contribute localisations for Geckium in your language via Geckium's GitHub page.">
\ No newline at end of file
+<!ENTITY gwizard.publicBetaTitle			"このバージョンの Geckium はベータ版です - 不安定かも？">

```

Comment:
I just translated it like "This version of Geckium is beta - Maybe unstable?", Should I need to translate it like the original en-us one?

Reply:
The string is not supposed to be a question, it's just informing the user that it may be unstable.
Something closer to the en-US one would be preferable :)

Reply:
little bit changed, is this translation fine now?

Reply:
Yes, thank you :)